resources/public/api/conf/4.0/brought_forward_losses_list.yaml

```diff
@@ -18,30 +18,26 @@ get:
               <td><p>N/A - DEFAULT</p></td>
               <td><p>Simulates success response with multiple business types.</p></td>
           </tr>
-      </tbody>
-      <tbody>
           <tr>
               <td><p>SELF_EMPLOYMENT</p></td>
               <td><p>Simulates success response with only self-employment.</p></td>
           </tr>
-      </tbody>
-      <tbody>
           <tr>
               <td><p>UK_PROPERTY</p></td>
               <td><p>Simulates success response with only UK property.</p></td>
           </tr>
-      </tbody>
-      <tbody>
           <tr>
               <td><p>FOREIGN_PROPERTY</p></td>
               <td><p>Simulates success response with only foreign property.</p></td>
           </tr>
-      </tbody>
-      <tbody>
           <tr>
               <td><p>NOT_FOUND</p></td>
               <td><p>Simulates the scenario where no data is found.</p></td>
           </tr>
+          <tr>
+              <td><p>STATEFUL</p></td>
+              <td><p>Performs a stateful retrieve.</p></td>
```

Comment:
I think going forward we are using `Performs a stateful list` for list endpoints!